data/sql_setup/prepare_schedule_e.sql

```diff
@@ -217,7 +217,7 @@ end
 $$ language plpgsql;
 
 drop trigger if exists ofec_sched_e_queue_trigger on fec_vsum_sched_e;
-create trigger ofec_sched_e_queue_trigger before insert or update or delete
+create trigger ofec_sched_e_queue_trigger instead of insert or update or delete
```

Comment:
Oh, I wasn't aware that there was a difference in the case of tables vs. views with triggers in that sense.  I should read up more too, but I had glanced at the beginning of the [Postgres Trigger documentation](https://www.postgresql.org/docs/9.5/static/sql-createtrigger.html) for that info.